scripts/quests/jeuno/Wings_of_Gold.lua

```diff
@@ -0,0 +1,100 @@
+-----------------------------------
+-- Wings of GOld
+-----------------------------------
+-- Log ID: 3, Quest ID: 60
+-- Brutus: !gotoid 17776652
+-- UPPER_DELKFUTTS_TOWER Chest !gotoid 17424564
+-- MIDDLE_DELKFUTTS_TOWER: !gotoid 17420676
+-----------------------------------
+local quest = Quest:new(xi.questLog.JEUNO, xi.quest.id.jeuno.WINGS_OF_GOLD)
+
+quest.reward =
+{
+    fame     = 20,
+    fameArea = xi.fameArea.JEUNO,
+    item     = xi.item.BARBAROI_AXE,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == xi.questStatus.QUEST_AVAILABLE and
+                player:hasCompletedQuest(xi.questLog.JEUNO, xi.quest.id.jeuno.PATH_OF_THE_BEASTMASTER) and
+                player:getMainLvl() >= xi.settings.main.AF1_QUEST_LEVEL and
+                player:getMainJob() == xi.job.BST
+        end,
+
+        [xi.zone.UPPER_JEUNO] =
+        {
+            ['Brutus'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') == 1 then
```

Comment:
For clarity, I would suggest starting with a 0-value check, and reordering the condition.  That should most likely reduce the need for two in-line comments

Reply:
♻